Domain.Tests/Validators/SchoolValidatorTests.cs

```diff
@@ -0,0 +1,186 @@
+﻿using Domain.Validators;
+using FluentValidation.TestHelper;
+using Xunit;
+
+namespace Domain.Tests.Validators;
+
+public class SchoolValidatorTests
+{
+    private readonly SchoolValidator _validator;
+
+    public SchoolValidatorTests()
+    {
+        _validator = new SchoolValidator();
+    }
+
+    [Theory]
+    [InlineData("")]
+    [InlineData(" ")]
+    [InlineData(null)]
+    public void With_no_name(string name)
+    {
+        var model = new School { EstablishmentName = name };
+        var result = _validator.TestValidate(model);
+        result.ShouldHaveValidationErrorFor(x => x.EstablishmentName);
+    }
+
+    [Theory]
+    [InlineData(0)]
+    [InlineData(-1)]
+    [InlineData(9998)]
+    public void With_invalid_urn(int id)
+    {
+        var model = new School { Urn = id };
+        var result = _validator.TestValidate(model);
+        result.ShouldHaveValidationErrorFor(x => x.Urn);
+    }
+
+    [Theory]
+    [InlineData(1000)]
+    public void With_valid_urn(int id)
+    {
+        var model = new School { LocalAuthorityDistrictId = id };
+        var result = _validator.TestValidate(model);
+        result.ShouldNotHaveValidationErrorFor(x => x.LocalAuthorityDistrictId);
+    }
+
+    [Theory]
+    [InlineData("")]
+    [InlineData(" ")]
+    [InlineData(null)]
+    public void With_no_address(string address)
+    {
+        var model = new School { Address = address };
+        var result = _validator.TestValidate(model);
+        result.ShouldHaveValidationErrorFor(x => x.Address);
+    }
+
+    [Theory]
+    [InlineData(0)]
+    [InlineData(12)]
+    [InlineData(-1)]
+    public void With_invalid_establishmentTypeGroupId(int id)
+    {
+        var model = new School { EstablishmentTypeGroupId = id };
+        var result = _validator.TestValidate(model);
+        result.ShouldHaveValidationErrorFor(x => x.EstablishmentTypeGroupId);
+    }
+
+    [Theory]
+    [InlineData(1)]
```

Comment:
You can use something like `Enum.GetValues((typeof(EstablishmentTypeGroup)))` to loop through all the enum values rather than hard coding them like this. That way any new ones will automatically be tested. Same for other tests below